src/devices/openxrheadset/OpenXrInterface.cpp

```diff
@@ -1704,36 +1832,59 @@ void OpenXrInterface::getAllPoses(std::vector<NamedPoseVelocity> &additionalPose
         }
     }
 
+    numberOfPoses += 52;  // adding also fingers
     additionalPoses.resize(numberOfPoses);
 
     size_t poseIndex = 0;
 
     auto& head = additionalPoses[poseIndex];
     head.name = "openxr_head";
     head.pose = headPose();
-    head.velocity = headVelocity();
+	head.velocity = headVelocity();
+    head.filterType = PoseFilterType::NONE;
```

Comment:
Do you actually need this? If so, we might consider using an input parameter to turn the filter on or off